test/libsolidity/util/TestFileParser.cpp

```diff
@@ -127,6 +127,8 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 							tie(call.signature, lowLevelCall) = parseFunctionSignature();
 							if (lowLevelCall)
 								call.kind = FunctionCall::Kind::LowLevel;
+							if (isBuiltinFunction(call.signature))
```

Comment:
@mijovic Somehow you removed these lines. Strange that we did not have any isoltest builtin function tests in place. We should merge this PR soon.

Reply:
Hm, I am not sure how I did that... Does it makes sense to extract this out and add as separate PR to merge these back?

Reply:
Yes, makes sense. Just created #11304.